apps/state/lib/state/stops_on_route.ex

```diff
@@ -207,11 +223,18 @@ defmodule State.StopsOnRoute do
   defp stop_id_or_parent(%{parent_station: id}), do: id
 
   @spec merge_ids([stop_id_list]) :: stop_id_list
-  def merge_ids([]), do: []
+  @spec merge_ids([stop_id_list], [stop_id_list]) :: stop_id_list
+  def merge_ids(lists_of_ids, override_lists \\ [])
+
+  def merge_ids([], _), do: []
+
+  def merge_ids(lists_of_ids, override_lists) do
+    sorted_lists = Enum.sort_by(lists_of_ids, &list_merge_key/1, &>=/2)
+    # overrides should be short or empty, so putting that first with ++ is
+    # fine.
+    lists_with_overrides = override_lists ++ sorted_lists
```

Comment:
if we limit each (route,direction) to one override, as discussed above, this gets simpler.